data-serving/scripts/converters.py

```diff
@@ -343,14 +348,89 @@ def convert_outbreak_specifics(id: str, reported_market_exposure: str,
         normalized = convert_bool(lives_in_wuhan)
         if normalized is not None:
             outbreak_specifics['livesInWuhan'] = normalized
-    except (ValueError):
+    except ValueError:
         logging.warning(
             '[%s] [outbreakSpecifics[livesInWuhan]] invalid value %s', id,
             lives_in_wuhan)
 
     return outbreak_specifics if outbreak_specifics else None
 
 
+def convert_location_string(location_str: str) -> Dict[str, Any]:
+    if pd.isna(location_str) or location_str.lower() == 'no' or location_str.lower() == 'none':
+        return None
+    if type(location_str) is not str:
+        raise ValueError('location is not a string')
+    if ';' in location_str or ':' in location_str or location_str.count(',') > 1:
+        raise ValueError('location is a list')
+
+    # Remove common prefixes 'travelled to' and 'traveled to' and lowercase it.
+    normalized_location = re.sub(
+        r'(travelled|traveled)\sto\s(the\s*)?',
+        '', location_str.lower())
+
+    # First try to look up the location in a hard-coded map that accounts for
+    # most popular locations that the lookup (below) misses.
+    location = LOCATIONS.get(normalized_location)
+    if location is not None:
+        return location
+
+    # Next, attempt to look up the location in a countries database.
+    try:
+        country = pycountry.countries.lookup(normalized_location)
+        return {
+            'country': country.name
+        }
+    except:
```

Comment:
Do we want to specify what we're except'ing, here?

Reply:
done (LookupError)